CMakeLists.txt

```diff
@@ -198,7 +207,7 @@ elseif(US_SDK_INSTALL_COMPONENT)
 endif()
 
 us_cache_var(BUILD_SHARED_LIBS ON BOOL "Build shared libraries")
-us_cache_var(CMAKE_DEBUG_POSTFIX d STRING "Executable and library debug name postfix" ADVANCED)
+#us_cache_var(CMAKE_DEBUG_POSTFIX d STRING "Executable and library debug name postfix" ADVANCED)
```

Comment:
does this mean we're doing away with building debug configurations with a "d" suffix for all binaries?

Reply:
Yes. I tried various things to leave it in, but couldn't get a bunch of newly ported tests to work properly. The tests in question load the bundle by name, and in debug builds need the "d", but not in release builds. I tried various things to initialize a variable in the C++ code with a suffix depending on the build type but was never quite successful.